src/muya/lib/index.js

```diff
@@ -50,13 +50,18 @@ class Muya {
       this.eventCenter.dispatch('muya-format-picker', { reference: null })
 
       // Commit native cursor position because right-clicking doesn't update the cursor postion.
-      const cursor = selection.getCursorRange()
-      this.contentState.cursor = cursor
+      const { start } = selection.getCursorRange()
+      // I don't know why the start.key is not equal end.key when I `right-click` in table cell.
+      // So I set the cursor at `start` -- @jocs
+      this.contentState.cursor = {
```

Comment:
Chromium selects the whole word if you right click on a valid word.

Reply:
>Chromium selects the whole word if you right click on a valid word.

 Select the whole word, but why they are in different paragraphs(two cells)?

Reply:
To be honest, I don't know.